truffle/src/main/ruby/core/rubinius/alpha.rb

```diff
@@ -43,3 +43,13 @@ def include(mod)
   end
 
 end
+
+module Kernel
+
+  # Rubinius defines this method differently, using the :object_class primitive.  The two primitives are very similar,
```

Comment:
`:vm_object_class` takes the object to get the class for.  `:object_class` always uses `self`.